app/helpers/link_helper.rb

```diff
@@ -0,0 +1,23 @@
+module LinkHelper
+
+  def link_to_new_group_ling ling, depth, options={}, &block
+    if can?(:create, ling)
+      link_to new_group_ling_path(current_group, :depth => depth), options, &block
+    else
+      options[:onclick] = "$('#create_ling_modal').modal('show');"
+      link_to "#", options, &block
+    end
+  end
+
+  def link_to_delete_ling ling, options={}, &block
```

Comment:
this method changes the "delete ling" link.
If the user can't destroy a ling and clicks on this link, a modal will appear and it shows a message: "Contact the admin to delete this ling"